core/domain/stats_jobs_one_off.py

```diff
@@ -1269,12 +1241,6 @@ def reduce(key, stringified_values):
                     'state hit count not same exp_id:%s state:%s, '
                     'all:%s sum: null' % (
                         key, state_name, all_state_hit[state_name]),)
-            elif all_state_hit[state_name] != sum_state_hit[state_name]:
```

Comment:
Explanation?

Reply:
Sorry, forgot about this one. `all_state_hit` gets updated when the version is 'all' else `sum_state_hit` gets updated. These two variables cannot be updated at the same time and hence this elif condition can never be True. This is because `sum_state_hit` would only contain zeroes as it won't get updated and `all_state_hit[state_name]` have to be zero for the if condition to fail